render_delegate/nodes/driver_main.cpp

```diff
@@ -113,18 +113,25 @@ driver_process_bucket
     ids.clear();
     const void* colorData = nullptr;
     const void* positionData = nullptr;
+    auto checkOutputName = [&outputName](const AtString& name) -> bool {
+#if ARNOLD_VERSION_NUMBER > 60201
+        return outputName == name;
+#else
+        return strcmp(outputName, name.c_str()) == 0;
```

Comment:
I think you might be able to do the following simplification since `AtString` is usually automatically converted to `char*`:
```
        return strcmp(outputName, name) == 0;

Reply:
Good point, thanks for the advice! Changed.